mango-python/bdgenomics/mango/QC.py

```diff
@@ -219,3 +221,136 @@ def plot(self, xScaleLog = False, yScaleLog = False, testMode = False, plotType=
             plt.show()
 
         return chromosomes
+
+class VariantEffectAlleleFreq(object):
+    """
+    todo: this needs to be modified to calculate bins in Spark so as to reduce Collect size
+    """
+    def __init__(self, ss, variantRDD):
+        so_terms = [
```

Comment:
Is there a different way to get these terms? can we pull them from avro or the rdd?

Reply:
@heuermh Are the Go Terms already listed somewhere?

Reply:
The best place to look for these is the VCF ANN Specification:
http://snpeff.sourceforge.net/VCFannotationformat_v1.0.pdf